app/src/main/java/com/github/se/cyrcle/model/parking/Parking.kt

```diff
@@ -47,6 +56,16 @@ enum class ParkingCapacity(override val description: String) : ParkingAttribute
   XLARGE("More than 100 spots")
 }
 
+// Reasons for reporting a Parking
+enum class ParkingReportReason(override val description: String, val severity: Int) :
+    DropDownableEnum {
+  POOR_MAINTENANCE("This Spot has Poor Maintenance", 1),
+  INACCESSIBLE("This Spot is Inaccessible", 2),
+  ILLEGAL_SPOT("This Spot is in an Illegal Place", 2),
+  SAFETY_CONCERN("This Spot is in an Dangerous Place", 3),
+  INEXISTANT("This Spot does Not Exist", 3)
```

Comment:
Nothing too important but isnt ILLEGAL_SPOT more severe than INEXISTANT ?

Reply:
I think they should both be max severity level. If it doesn't even exist, it shouldn't be in the app!

Reply:
No i think it's as bad as inaccessible. An inexistant spot is inaccessible. Safety concerns and legal reasons are on whole other level.